x-pack/plugins/stack_connectors/server/connector_types/stack/opsgenie/schema.ts

```diff
@@ -15,15 +17,64 @@ export const SecretsSchema = schema.object({
   apiKey: schema.string(),
 });
 
+const SuccessfulResponse = schema.object(
+  {
+    took: schema.number(),
+    requestId: schema.string(),
+    result: schema.string(),
+  },
+  { unknowns: 'allow' }
+);
+
+export const FailureResponse = schema.object(
+  {
+    took: schema.number(),
+    requestId: schema.string(),
+    message: schema.maybe(schema.string()),
+    result: schema.maybe(schema.string()),
+    /**
+     * When testing invalid requests with Opsgenie the response seems to take the form:
+     * {
+     *   ['field that is invalid']: 'message about what the issue is'
+     * }
+     *
+     * e.g.
+     *
+     * {
+     *   "message": "Message can not be empty.",
+     *   "username": "must be a well-formed email address"
+     * }
+     */
+    errors: schema.maybe(schema.any()),
```

Comment:
Do you think changing it to `Record<string, string>` will be risky? If the error schema is different it will throw an error.

Reply:
Yeah that was my concern, I figured the safest route is just to `JSON.stringify` the result and not actually make assumptions on how it is formatted. When I looked through the Opsgenie API docs I didn't see any references to the format of the `errors` object. So my understanding of the format is just based on my experience with trying to cause errors.

Reply:
Ok, make sense!